src/actions/action.ts

```diff
@@ -36,7 +36,7 @@ export interface ActionSpec {
    * Can be defined to display an icon
    */
```

Comment:
Is it just me, or is the top/bottom padding of the menu smaller than the one of the other menus ?